zgrab_schema.py

```diff
@@ -800,29 +805,23 @@
                     "compression":String(),
                 }),
             }),
-            "dh_key_exchange": SubRecord({
-                "params": SubRecord({
-                    "prime": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "generator": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_private": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
+            "key_exchange": SubRecord({
+                "ed25519sha256_params": SubRecord({
```

Comment:
Not AFAIK. But this key exchange is referred to internally (by the x/ssh code) as `curve25519sha256`, and I just carried it forward here.